bin/install-qa-check.d/60openrc

```diff
@@ -33,6 +33,21 @@ openrc_check() {
 			done
 		done
 	fi
+
+	# Make sure deprecated #!/sbin/runscript isn't being used:
+	if [[ ! -d "${ED}"/etc/init.d ]] ; then
```

Comment:
Again, it's done earlier in the file (and more clear IMO to match other uses of the same variable).